Mathlib/Util/Delaborators.lean

```diff
@@ -149,7 +149,8 @@ def exists_delab : Delab := whenPPOption Lean.getPPNotation do
       if x == y then `(∃ $x:ident ⊃ $z, $body) else pure stx
     | _ => pure stx
   match stx with
-  | `(∃ $group:bracketedExplicitBinders, ∃ $groups*, $body) => `(∃ $group $groups*, $body)
+  | `(∃ $group:bracketedExplicitBinders, ∃ $[$groups:bracketedExplicitBinders]*, $body) =>
```

Comment:
The local context makes it seem like `$[$groups:bracketedExplicitBinders]*` is not necessary (it says `groups` is a ``TSyntaxArray `Lean.bracketedExplicitBinders``), but I guess the matcher it generates without the explicit annotation is too general.